terraform/compass_module/main.tf

```diff
@@ -15,46 +11,4 @@ terraform {
 
 provider "aws" {
   region  = var.aws_region
-  profile = var.aws_user
```

Comment:
needed to remove this to be able to run, looks like there's other ways to set this (e.x. through an environment variable)